src/qs8-vmulc/rvv.c.in

```diff
@@ -0,0 +1,69 @@
+// Copyright 2023 Google LLC
+//
+// This source code is licensed under the BSD-style license found in the
+// LICENSE file in the root directory of this source tree.
+
+#include <assert.h>
+#include <riscv_vector.h>
+#include <xnnpack/vbinary.h>
+
+$assert DATATYPE in ["QS8", "QU8"]
+$XINT8_T = {"QS8": "int8_t", "QU8": "uint8_t"}[DATATYPE]
+$VSIZE = {"f2": "mf2", 1: "m1", 2: "m2"}[LMUL]
+$VSIZE2 = {"f2": "m1", 1: "m2", 2: "m4"}[LMUL]
+$VSIZE4 = {"f2": "m2", 1: "m4", 2: "m8"}[LMUL]
+
+void xnn_${DATATYPE.lower()}_vmulc_minmax_fp32_ukernel__rvv_u${BATCH_TILE}(
```

Comment:
Question: 
Should we use `{BATCH_TILE}` in the function name?
or we can just imitate existing ukernels now.

Based on my understanding, `{BATCH_TILE}` means how many elements are processed in a single loop.
However, RVV's VLEN is nondeterministic and depends on the running machine. 
If we use `{BATCH_TILE}` here, it means that we have the expected VLEN size. I saw existing kernels(e.g hswish, vsqrt) use `uX` to correspond to 'LMUL=X'. This is a little bit different from other platforms, but it is fast for developers to integrate ukernels without modifying the python script.

Reply:
@alankelly Could you provide any suggestions?

Reply:
Can you do like hswish with u${LMUL}v so instead of the number of elements the loop is unrolled by use the number of vectors?

https://github.com/google/XNNPACK/blob/master/src/f32-vhswish/gen/f32-vhswish-rvv-u2v.c

Reply:
Changes done and PR updated accordingly. Thanks.